blocks/form/form.js

```diff
@@ -1,3 +1,5 @@
+import formatFns from './formatting.js';
```

Comment:
good idea to extract the formatter functions. To make this a bit more robust, I'd try to make the import `async` using `import()` and have a fallback.

Reply:
resolved https://github.com/vdua/nedbank/pull/1/files#diff-14d0faad899dcfcb4e5cd318658f4d58b9e2aa66ebc5912bbe09302339f6a564R1